sts-spring/src/main/java/de/adorsys/sts/token/tokenexchange/server/TokenExchangeController.java

```diff
@@ -127,7 +60,7 @@ public ResponseEntity<Object> tokenExchange(
             ResponseEntity<Object> errorData = ResponseUtils.invalidParam(e.getMessage());
             return ResponseEntity.badRequest().body(errorData);
         } finally {
-            if (logger.isTraceEnabled()) logger.trace("POST tokenExchange finished.");
+            if (log.isTraceEnabled()) log.trace("POST tokenExchange finished.");
```

Comment:
This block is always executed, which leads to a wrong log message in case there was an exception (Should log an exception in case there was one)